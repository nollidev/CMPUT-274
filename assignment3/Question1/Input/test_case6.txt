26 5
W N C F V X G D O H Z B A T E M R Q I P J Y K U L S
