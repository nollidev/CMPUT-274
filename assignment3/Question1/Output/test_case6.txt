W -> N C F V X
