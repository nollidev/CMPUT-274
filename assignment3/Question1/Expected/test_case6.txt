W -> N C F V X
N -> G D O H Z
C -> B A T E M
F -> R Q I P J
V -> Y K U L S
